composer.json

```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": ">=5.0.0"
+        "php": ">=5.6"
```

Comment:
Is it proper to support `php-5.6` version at least?

The `php-5.6` and `php-7.0` are unmaintained on official PHP team.

And the `php-7.0` has been release for about three years.

If we want to keep the package e compatibility, I think we can let the package require `php-7.0` version at least.

Reply:
See? I debated that, too...

I figured if it was going from, literally, 5.0.0, maybe it'll be too much of a shock to active users, especially on legacy sites? Especially since this isn't even composer friendly...

Reply:
OK but. I just started thinking... since this will be in effect a new project, I officially hard forked it to phpexpertsinc/chromephp and will be releasing a new composer package. It will definitely target PHP 7.0.

Reply:
Maybe we can release two versions.

One is about supporting PHP 5.6.

Another one is about supporting PHP 7.1+.

This can be compatible with PHP 5 users.

Reply:
That's right.

I think releasing the new version is the proper way because original package has been inactive for about long time.